src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAroundCheck.java

```diff
@@ -389,6 +389,7 @@ public int[] getAcceptableTokens() {
             TokenTypes.GENERIC_START,
             TokenTypes.GENERIC_END,
             TokenTypes.ELLIPSIS,
+            TokenTypes.LITERAL_WHEN,
```

Comment:
@nrmancuso Is there a reason we don't make this default? We should our reasoning to the first post in the issue.

It seems like this won't hurt anything unless its used and if it is used, it would only be on a project which supports it. I am not seeing any negatives yet on why this can't be defaulted.

Reply:
Good point, let’s make this default

Reply:
@mahfouz72 Please make this default also. This should also revert `checkstyle-checks.xml` to what it was before.

Reply:
done added to the default tokens and revered checkstyle-checks.xml